doc/setup.md

```diff
@@ -165,3 +165,18 @@ make load_demo_data
 ```
 
 This also adds `testuser@cesium-ml.org` as an administrator.
+
+### nginx configuration
+
+When running on a public server, it can be useful to use X. 509 certificates for Transport Layer Security (TLS) encryption. For example, services such as Let's Encrypt (https://letsencrypt.org/) provide free services to create such certificates when deploying publicly.
+
+One way to pick up the certificates with SkyPortal's nginx deployment is by modifying the default nginx template configuration files which come with baselayer. In this case, in the server portion of baselayer/services/nginx/nginx.conf.template, one can add (for Let's Encrypt'):
+
+    listen [::]:443 ssl ipv6only=on;
```

Comment:
Why ipv6only?

Reply:
This was just what Let's Encrypt added automatically... I am not sure if I know enough to talk intelligently here :)